<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ViewSB</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="1" column="1">
     <widget class="QTreeWidget" name="usb_tree_widget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>9</number>
      </property>
      <column>
       <property name="text">
        <string>summary</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>timestamp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dev</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ep</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dir</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>length</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>packet#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>data</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="usb_details_dock">
   <property name="windowTitle">
    <string>Details</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="usb_details_dock_contents">
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="ViewSBQTreeWidget" name="usb_details_tree_widget">
       <property name="initialSize" stdset="0">
        <size>
         <width>550</width>
         <height>200</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>property</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>value</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="hex_dock">
   <property name="windowTitle">
    <string>Raw</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="hex_dock_contents">
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="ViewSBHexView" name="usb_hex_view">
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewSBQTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>qt.py</header>
  </customwidget>
  <customwidget>
   <class>ViewSBHexView</class>
   <extends>QTableView</extends>
   <header>qt.py</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
